lms/static/scripts/frontend_apps/components/LaunchErrorDialog.tsx

```diff
@@ -657,6 +657,28 @@ export default function LaunchErrorDialog({
         </ErrorModal>
       );
 
+    case 'canvas_submission_max_attempts':
+      return (
+        <ErrorModal
+          {...defaultProps}
+          onRetry={undefined}
+          title="Grading submission failed"
+        >
+          <p>
+            Your annotation was saved, but Hypothesis could not record a new
+            submission for grading. Hypothesis has already recorded the maximum
+            number of allowed submissions for this assignment.
+          </p>
+          <p>
+            See{' '}
```

Comment:
Should we include this link in the other submission related errors? 

It could be done in a follow up PR if we decide to include them.

Reply:
Hmm... maybe. I can discuss with others.